src/components/Navbar/NavItems.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import React from 'react';
+import { Divider } from '@material-ui/core';
+import { User, FileText, Folder, Type, MessageCircle } from 'react-feather';
+import { NavItem, NavItemProps } from '../NavItem';
+import { CFC } from '../../typings/components';
+
+type NavItemsProps = {
+  onClick?: NavItemProps['onClick'];
+};
+
+const NavItems: CFC<NavItemsProps> = ({ onClick }) => {
+  return (
+    <>
+      <NavItem to="/about" onClick={onClick} icon={User}>
+        About
+      </NavItem>
+      <Divider flexItem orientation="vertical" />
+      <NavItem to="/resume" onClick={onClick} icon={FileText}>
+        Resume
+      </NavItem>
+      <Divider flexItem orientation="vertical" />
+      <NavItem to="/portfolio" onClick={onClick} icon={Folder}>
+        Portfolio
+      </NavItem>
+      <Divider flexItem orientation="vertical" />
+      <NavItem to="/blog" onClick={onClick} icon={Type}>
```

Comment:
currently named `portfolio-page` so we should change the route or page name :D

Reply:
Let's move this conversation to https://github.com/CodersCrew/CodersCard/pull/73#discussion_r486573523